woocommerce/woocommerce-android
# Add Jetpack site stats to the Analytics Hub

Summary
==========
Fix issue #7574 by implementing the basic structure of the Visitors and Views card.

This PR aggregate the following tasks:

- [x] Basic card UI (this PR)
- [x] Basic Data fetching (this PR)
- [x] Day, week, month and year date range selection (this PR)
- [x] Quarter date range selection (https://github.com/woocommerce/woocommerce-android/issues/7750)
- [x] Hide Visitors and Stats when Custom date range is selected (https://github.com/woocommerce/woocommerce-android/issues/7780)
- [x] Delta percentage badge info (https://github.com/woocommerce/woocommerce-android/issues/7751)

Screenshots
==========
https://user-images.githubusercontent.com/5920403/200482962-38d59ddf-c613-4b56-85ae-0886ec886dbb.mp4

How to Test
==========
Before testing, please bear in mind that the implementation introduced here still doesn't properly support the `Custom` and `Quarter` date range selection for the Visitors and Views card.

1. Go to your store jetpack stats (URL should be <your-store-address>/wp-admin/admin.php?page=stats)
2. Verify the data there for days, weeks, months and years granularity
3. Check if the data displayed in the Visitors and Views card in the Analytics Hub corresponds to the data displayed in your store stats.

Update release notes:
